hw/ip/i2c/data/i2c.hjson

```diff
@@ -133,6 +133,16 @@
       type: int
       default: "268",
     }
+    { name:    "InputDelayCycles",
+      type:    "int",
+      default: "0",
+      desc:    '''
+               Maximum number of cycles of propagation delay between a change on the cio_scl_en_o or cio_sda_en_o pins and sensing the new values on the corresponding input pins, not including the rise/fall times.
+               For the purposes of this calculation, an input delay of 0 cycles means an output pin changing at the beginning of clock edge N will be sampled and observed on the input pins at clock edge N+1.
+               '''
+      local:   "false",
+      expose:  "true"
+    }
```

Comment:
This change should already be in the first commit, right? Because the parameter is referred to there

Reply:
Sure, that works. Squashed.